.editorconfig

```diff
@@ -0,0 +1,13 @@
+# top-most EditorConfig file
+root = true
+
+# Unix-style newlines with a newline ending every file
+[*]
+end_of_line = lf
+insert_final_newline = true
+indent_style = space
+indent_size = 4
+charset = utf-8
+
+[*.xml]
+indent_size = 2
```

Comment:
Not against this.  Useful for IntelliJ? Possibly move to separate PR along with `.gitignore`

Reply:
This is used by mose IDEs nowadays, IntelliJ, VSCode and Eclipse (probably with a Plugin( at least. Which IDE are you using?